pkg/steps/template.go

```diff
@@ -38,7 +38,8 @@ type templateExecutionStep struct {
 }
 
 const (
-	showOutputAnnotation string = "ci-operator.openshift.io/always-show-output"
+	showOutputAnnotation          string = "ci-operator.openshift.io/always-show-output"
```

Comment:
This annotation should be removed after we transition all existing users to the new one.